docs/machine-learning/snippets/mldotnet-api/csharp/Program.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+class Program
+{
+    public class HouseData
+    {
+        public float Size { get; set; }
+        public float Price { get; set; }
+    }
+
+    public class Prediction
+    {
+        [ColumnName("Score")]
+        public float Price { get; set; }
+    }
+
+    static void Main(string[] args)
+    {
+        MLContext mlContext = new();
+
+        // 1. Import or create training data.
+        HouseData[] houseData = [
+                new HouseData() { Size = 1.1F, Price = 1.2F },
+                new HouseData() { Size = 1.9F, Price = 2.3F },
+                new HouseData() { Size = 2.8F, Price = 3.0F },
+                new HouseData() { Size = 3.4F, Price = 3.7F }
+                ];
```

Comment:
This could be simplified, if you want:

```suggestion
        HouseData[] houseData = [
                new () { Size = 1.1F, Price = 1.2F },
                new () { Size = 1.9F, Price = 2.3F },
                new () { Size = 2.8F, Price = 3.0F },
                new () { Size = 3.4F, Price = 3.7F }
                ];
```